src/components/calcite-action-pad/calcite-action-pad.e2e.ts

```diff
@@ -1,18 +1,108 @@
+import { newE2EPage } from "@stencil/core/testing";
 import { accessible, hidden, renders } from "../../tests/commonTests";
+import { setUpPage } from "../../tests/utils";
 
 describe("calcite-action-pad", () => {
   it("renders", async () => renders("calcite-action-pad"));
 
   it("honors hidden attribute", async () => hidden("calcite-action-pad"));
 
+  it("defaults", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent("<calcite-action-pad></calcite-action-pad>");
+    const element = await page.find("calcite-action-pad");
+    expect(element.getAttribute("expand")).not.toBeNull();
+    expect(element.getAttribute("expanded")).toBeNull();
+  });
+
+  it("expand: true", async () => {
```

Comment:
Nitpick: expand tests could be nested in a `describe` or handled in a single test (reasoning: tests the same behavior).